src/services/ui/src/pages/faq/content/oneMACFAQContent.tsx

```diff
@@ -155,6 +181,43 @@ export const oneMACFAQContent: FAQContent[] = [
           </section>
         ),
       },
+      {
+        anchorText: "onboarding-materials",
+        question: "Onboarding Materials",
+        answerJSX: (
+          <>
+            <ul>
+              {[
+                ["WelcometoOneMAC.pdf", "Welcome to OneMAC"],
+                [
+                  "IDMInstructionsforOneMACUsers.pdf",
+                  "IDM Instructions for OneMAC Users",
+                ],
+                ["OneMACIDMGuide.pdf", "OneMAC IDM Guide"],
+                ["OneMACStateUserGuide.pdf", "OneMAC State User Guide"],
+                ["OneMACCMSUserGuide.pdf", "OneMAC CMS User Guide"],
+              ].map(([filename, label]) => (
+                <li key={filename}>
+                  <a
+                   className="text-blue-800 underline hover:no-underline "
+                    // href={`${process.env.PUBLIC_URL}/assets/onboarding/${filename}`}
```

Comment:
if we need to go get these files and pul them in the assets folder, lets do that and remove the env value